### PR TITLE
Clock Operation to Serial Monitor

### PR DESCRIPTION
**Pull request by chrisyazar28**
I did the clock operation to be able to see it on the serial monitor and add 2 variables for the same operation.

### CODE REVIEW

File: tix_clock/tix_clock.ino [hunk 1]
```diff
@@ -1,12 +1,14 @@
 const int MINUTES_BUTTON_PIN = 5;
 const int HOURS_BUTTON_PIN = 6;
+const int MODIFY_HOUR_PIN = 7;
 const int NUMBER_OF_INPUT_PINS = 2;
```
Comment (caralvar) [inline tix_clock/tix_clock.ino]: If we have now 3 inputs, please increment the number of inputs to 3

File: tix_clock/tix_clock.ino [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 int INPUT_PINS[] = {MINUTES_BUTTON_PIN,HOURS_BUTTON_PIN};
```
Comment (caralvar) [inline tix_clock/tix_clock.ino]: Add the new input to the input pins list

**Review by caralvar — CHANGES_REQUESTED** [review]

File: tix_clock/tix_clock.ino (hunk 1 above)
Comment (chrisyazar28) [reply tix_clock/tix_clock.ino]: I already increased the inputs

File: tix_clock/tix_clock.ino (hunk 2 above)
Comment (chrisyazar28) [reply tix_clock/tix_clock.ino]: Is already added in the list.

**Pull request merged**